src/main/java/sirius/db/mixing/types/MultiLanguageString.java

```diff
@@ -67,7 +67,7 @@ public MultiLanguageString addText(String text) {
     /**
      * Adds a new text for the given language.
      * <p>
-     * Null texts will be ignored.
+     * Texts with null values will be removed.
```

Comment:
there are other methods that call this method which have the same javadoc which needs to be adjusted